### PR TITLE
Add releaser action and pre-commit config

### PR DESCRIPTION
**Pull request by trungleduc**

### CODE REVIEW

**Comment by trungleduc** [comment]
7nzqv6

**Comment by martinRenou** [comment]
What! The bot is talking 😮

**Comment by trungleduc** [comment]
> Just created the access tokens with the bot account!
> 
> Let's merge and try it!

I need to wait for the check release job, it failed in the previous run.

File: .github/workflows/check-release.yml [hunk 1]
```diff
@@ -1,40 +1,28 @@
...
-
+      - name: Install Dependencies
+        run: |
+          pip install .
       - name: Check Release
-        if: ${{ matrix.group == 'check_release' }}
         uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          version_spec: next
```
Comment (martinRenou) [inline .github/workflows/check-release.yml:33]: Could this be the reason for the failure? We probably need to keep this version_spec value?

**Pull request merged**